### PR TITLE
Ports - Jillianne

### PR DESCRIPTION
**Pull request by jillirami**
# JS Adagrams
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
--- | ---
What patterns were you able to use from your Ruby knowledge to apply to JavaScript? | I was able to use loops, hashes and arrays to hold and access information similarly to how I would/did in Ruby.
Did you need to use different strategies to find information online about JavaScript than you would for Ruby? | Other than including JavaScript in my search, there is not much that is different.
What was a challenge you were able to overcome on this assignment? | I was able to work through syntax errors to get my code to function.
What has been interesting and positive about learning a new programming language? | I feel more confident in my ability to succeed in an internship or future job, that I am able to learn another programming language.
What is something to focus on your learnings in JavaScript in the next week? | I will continue to focus on manipulating data structures and enhancing my logic.

### CODE REVIEW

File: src/adagrams.js [hunk 1]
```diff
@@ -1,8 +1,150 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+class Adagrams {
+  static drawLetters() {
+    const availableHand = []
+    for (let i = 0; i < 9; i++) {
```
Comment (CheezItMan) [inline src/adagrams.js:4]: These loops seem a little awkward.  Is there a more elegant way to do this?

**Comment by CheezItMan** [comment]
# JS Adagrams
## What We're Looking For



Feature | Feedback
--- | ---
**General** |
Answered comprehension questions |  Check, I'm glad you feel more confident now.  
Small commits with meaningful commit messages |  Check, but avoid mention of waves in your commit messages.  Instead focus on functionality you've added. 
**Code Requirements** |
`drawLetters` method |  Check
Uses appropriate data structure to store the letter distribution |  Check
All tests for `drawLetters` pass |  Check
`usesAvailableLetters` method |  Check
All tests for `usesAvailableLetters` pass |  Check
`scoreWord` method |  Check
Uses appropriate data structure to store the letter scores |  Check
All tests for `scoreWord` pass |  Check
`highestScoreFrom` method |  Check
Appropriately handles edge cases for tie-breaking logic |  Check
All tests for `highestScoreFrom` pass |  Check
**Overall** |  Nice work, you hit all the learning goals and even got the class working!
